frontend/src/metabase-lib/types.ts

```diff
@@ -382,17 +382,29 @@ export type StringFilterOptions = {
 
 export type NumberFilterValue = number | bigint;
 
+export type NumberFilterOptions = {
+  minInclusive?: boolean;
+  maxInclusive?: boolean;
```

Comment:
Very nitty pick, but I think the imperative mood is better suited for options here:

```suggestion
  includeMin?: boolean;
  includeMax?: boolean;
```